index.html

```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
         </ul>
       </div>
     </section>
+    <section id="portfolio">
+      <div class="title">
+        <h2>My Recent Works</h2>
+        <hr />
+      </div>
+      <div class="plchdr">
+        <div class="img">
+          <img src="/images/plc-img.jpg" alt="" />
```

Comment:
- [ ] As the previous reviewer mentioned, please know that you should leave the alt attributes empty for your images. You should add a descriptive alt attribute to your images. 🙏